input/growth/deploy_and_manager_insts.xml

```diff
@@ -0,0 +1,109 @@
+<simulation>
+  <control>
+    <duration>10</duration>
+    <startmonth>1</startmonth>
+    <startyear>2018</startyear>
+    <decay>never</decay>
+  </control>
+
+  <archetypes>
+    <spec> <lib>cycamore</lib><name>GrowthRegion</name> </spec>
+    <spec> <lib>cycamore</lib><name>DeployInst</name> </spec>
+    <spec> <lib>cycamore</lib><name>ManagerInst</name> </spec>
+    <spec> <lib>cycamore</lib><name>Source</name> </spec>
+    <spec> <lib>cycamore</lib><name>Sink</name> </spec>
+  </archetypes>
+
+  <commodity>
```

Comment:
I'm not sure this block is even needed for this simple simulation. At least, upon skimming the resulting DBs with and without it, I didn't see a difference. We can get someone more knowledgable to weigh in.

Reply:
I felt like it was good practice to include a sink in the simulation. Also, this way it matches more with the `source_sink_linear.xml` file also in that directory.

Reply:
Oh I meant the commodity block with the priority, not the sink. I should have done a better job w that comment :-)

Reply:
(this is a comment of the lowest priority, btw)

Reply:
That was just a force of habit. If you really think it needs to be removed I can do that.

Reply:
It depends if we prefer the input file to be a good example or to be bare bones. I was reviewing from the bare bone perspective, which given the remaining questions in Issue #530 may or may not be valid.

Reply:
This is definitely not a necessary block, but I would NOT require its remove for this to merge.